apps/frontend/src/components/fap/Proposals/FapAssignedReviewersTable.tsx

```diff
@@ -184,15 +184,15 @@ const FapAssignedReviewersTable = ({
           onClose={() => setRankReviewer(null)}
           onSubmit={async (value) => {
             await api().saveReviewerRank({
-              proposalPk: rankReviewer?.proposal as number,
+              fapReviewId: rankReviewer?.fapReviewId as number,
               reviewerId: rankReviewer?.reviewer as number,
```

Comment:
```suggestion
              reviewerId: rankReviewer.reviewer as number,
```
Optional chaining not needed also here. I do realize you did not introduce this change, but if you know why we are certain the `rankReviewer.reviewer` is not null would be nice to add comment.